src/Modules/Backend/assets/Backend/webpack/scss/components/_cards.scss

```diff
@@ -135,7 +131,7 @@
       color: $body-color;
     }
 
-    .inputCheckbox  + .card-title {
+    .input-checkbox  + .card-title {
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. 
Let's focus here on the CI issue, otherwise this PR will get out of focus.